command/s3_put.go

```diff
@@ -137,6 +140,9 @@ func (s3pc *s3put) validate() error {
 	if s3pc.isMulti() && filepath.IsAbs(s3pc.LocalFile) {
 		catcher.Add(errors.New("cannot use absolute path with local_files_include_filter"))
 	}
+	if s3pc.Visibility == "signed" && (s3pc.Permissions == "public-read" || s3pc.Permissions == "public-read-write") {
```

Comment:
should we use the AWS SDK's constants (similar to [here](https://github.com/evergreen-ci/evergreen/blob/ea6d818a55e66067609f812947b74a9e6a52a227/command/s3_util.go#L58))?